statsmodels/stats/anova.py

```diff
@@ -293,7 +293,32 @@ def anova_lm(*args, **kwargs):
     Returns
     -------
     anova : DataFrame
-    A DataFrame containing.
+        When args is a single model, DataFrame contains.
+        sum_sq : float64
+            Sum of squares for model terms.
+        df : float64
+            Degress of freedom for model terms.
+        test statistic : float64
+            Name dependens on selected test statistic. Returns test statistic
```

Comment:
maybe something
The column name is either "F" or "chi2" depending on whether the F-statistic or the chisquare-statistic is used in computing the p-value.
I don't have an example right now to check how chisquare is spelled in the column name.

Reply:
Possible values for test are as follows:
test : str {"F", "Chisq", "Cp"} or None

Value of the statistic column and Pvalue column return exactly those inputs.

Possible Alternative for docstring:
...
{'F','Chisq','CP'} : float64
....
PR(>{'F','Chisq','CP'}) : float64

Reply:
I briefly browsed the code and it seems to me only "F" is implemented. (I don't even know what Cp stands for.

I would also prefer if the header/title for the column in the docstring were the same as the pandas column name. But none of the formats look really convincing to me and I don't have much of a preference.

PR(>{'F','Chisq','CP'}) : float64
looks more difficult to interpret than the full list, althout it's shorter, e.g.
`PR(>F),  P(>Chisq) or P(>CP)`

Reply:
You are correct...You can enter the Chisq or CP but it only returns 0.0 because only the 'F' actually makes anything calculate.  (no errors or warnings are returned)  Only clue is that P-value and test statistic is 0.0.

CP probably refers to Mallows Cp (never heard of it until today):
http://rafalab.github.io/pages/754/section-09.pdf

I too like the easy to read version.  I will add a see notes in the parameters that 'F' is the only implemented test until that functionality can be added later vs removing that parameter from the input.

Reply:
I think it would be easier for now to just use the "F" version in the description of the DataFrame, which will be much easier.
In this case I don't see an advantage of documenting a return that never shows up in a meaningful way.

Reply:
Commit https://github.com/statsmodels/statsmodels/pull/5233/commits/c121983cda118364398fc79f16d615eb2eb65dfe should finish these up